composer.json

```diff
@@ -51,6 +51,12 @@
     "conflict": {
         "symfony/symfony": "*"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/theofidry/AliceBundle"
+        }
+    ],
```

Comment:
Why this is needed now?

Reply:
Oh yes, I forgot to explain it in the PR description. It is required due to this unfortunate "incident": https://github.com/nelmio/alice/issues/1089